src/utils/Label.js

```diff
@@ -5,18 +5,20 @@ import tail from "lodash/tail";
 import min from "lodash/min";
 import max from "lodash/max";
 import reduce from "lodash/reduce";
-import numeral from "numeral";
 import { timeFormat } from "d3";
+import { format as numberFormat } from "d3-format";
 
 export function getDefaultFormats(scaleType) {
   const defaultTimeFormats = ["%Y", "'%y", "%b %Y", "%m/%Y"];
+  // Number format examples for a value of 1234.55555
+  // https://github.com/d3/d3-format
   const defaultNumberFormats = [
-    "0.[00]a",
-    "0,0",
-    "0.[0]",
-    "0.[00]",
-    "0.[0000]",
-    "0.[000000]"
+    "~s", // 1.23456k
```

Comment:
Thanks for commenting the output, super helpful. Inspired to do this for `defaultTimeFormats` now.